tests/conftest.py

```diff
@@ -18,7 +18,7 @@ def html_dir():
     docs_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'docs')
     build_path = os.path.join(docs_path, '_test', 'html')
     os.chdir(docs_path)
-    result = build_main(['sphinx-build', '-b', 'html', '.', build_path])
+    result = build_main(['-b', 'html', '.', build_path])
```

Comment:
https://github.com/sphinx-doc/sphinx/pull/3668/files

Reply:
In latest version first arg is not ignoring thats why it throws the error due to `'sphinx-build'`